extensionsGallery-insider.json

```diff
@@ -5511,6 +5511,67 @@
 					],
 					"statistics": [],
 					"flags": ""
+				},
+				{
+					"extensionId": "109",
+					"extensionName": "couchbase",
+					"displayName": "Couchbase",
+					"shortDescription": "Welcome to the official Azure Data Studio extension for Couchbase! This extension is designed to provide a seamless experience for Couchbase Server users who want to work within the popular Visual Studio Code editor.",
+					"publisher": {
+						"displayName": "Aayush Tyagi",
+						"publisherId": "AayushTyagi1",
+						"publisherName": "Aayush Tyagi"
+					},
+					"versions": [
+						{
+							"version": "2.1.4",
+							"lastUpdated": "09/03/2024",
+							"assetUri": "",
+							"fallbackAssetUri": "fallbackAssetUri",
+							"files": [
+								{
+									"assetType": "Microsoft.Couchbase.DownloadPage",
```

Comment:
This should be `"Microsoft.SQLOps.DownloadPage"`

Reply:
Thank Fixed